### PR TITLE
Add improvedigital to RTC callout vendors

### PR DESCRIPTION
**Pull request by mnishihan**

### CODE REVIEW

**Comment by mnishihan** [comment]
I strongly believe - the CircleCI tests, that are failing, are intermittent, caused by networking issues in CircleCI environment & unrelated with the changes in my PR.

For example, [this](https://app.circleci.com/pipelines/github/ampproject/amphtml/26332/workflows/31b1a052-5737-4cc9-96a4-81965bf9a65d/jobs/548565/parallel-runs/0/steps/0-118) & [this](https://app.circleci.com/pipelines/github/ampproject/amphtml/26332/workflows/31b1a052-5737-4cc9-96a4-81965bf9a65d/jobs/548565/parallel-runs/0/steps/0-118) failures are related to PGP key fetching from `keybase.io` and the [other one](https://app.circleci.com/pipelines/github/ampproject/amphtml/26332/workflows/31b1a052-5737-4cc9-96a4-81965bf9a65d/jobs/548568/parallel-runs/0/steps/0-113) also related to some XHR failing.

Also note that, yesterday I opened [another PR](https://github.com/ampproject/amphtml/pull/38484/) with almost identical changes and [all checks](https://github.com/ampproject/amphtml/pull/38484/checks) were passed.

Is there any way, I can rerun the failed tests, or anyone can do so??

TIA

File: extensions/amp-a4a/rtc-documentation.md [hunk 1]
```diff
@@ -153,6 +153,7 @@ The `errorReportingUrl` property is optional. The only available macros are ERRO
 -   Salesforce
 -   [Snigel](https://www.snigel.com)
 -   T13
+-   [Improve Digital](https://www.improvedigital.com)
```
Comment (powerivq) [inline extensions/amp-a4a/rtc-documentation.md:156]: Nit: order alphabetically

**Review by powerivq — CHANGES_REQUESTED** [review]

**Comment by powerivq** [comment]
As to the failing builds, you might want to try rebasing the latest main if that happens.

**Comment by mnishihan** [comment]
> As to the failing builds, you might want to try rebasing the latest main if that happens.

@powerivq Done.

File: extensions/amp-a4a/rtc-documentation.md (hunk 1 above)
Comment (mnishihan) [reply extensions/amp-a4a/rtc-documentation.md]: Done

**Review by powerivq — APPROVED** [review]

**Pull request merged**